stdlib/public/core/Integers.swift.gyb

```diff
@@ -2171,6 +2172,7 @@ extension FixedWidthInteger {
   
   // Homogeneous masking shift
 ${operatorComment(x.operator, False)}
+  @_semantics("optimize.sil.specialize.generic.partial.never")
```

Comment:
@slavapestov It is a very good question. I see the following possibilities for not-inlining transparent functions during the mandatory inlining:
- mutually recursive transparent functions
- the calls of protocol methods cannot be inlined until they are specialized/devirtualized later, so that a concrete @transparent method being called is not known during the mandatory inlining.

As a result, we process such calls of transparent functions later, when we perform the non-mandatary optimizations.